app/services/meeting_notifications.rb

```diff
@@ -1,27 +1,29 @@
 # frozen_string_literal: true
 
-module Notifications
-  extend ActiveSupport::Concern
+class MeetingNotifications
+  attr_reader :current_user
+  attr_reader :meeting
+  attr_reader :type
+  attr_reader :members
+  attr_reader :uniqueid
 
-  included do
-    helper_method :notifications_for_meeting_members,
-                  :delete_comment_and_notifications
+  def initialize(args)
+    @current_user = args[:current_user]
+    @meeting = args[:meeting]
+    @type = args[:type]
+    @members = args[:members]
+    @uniqueid = "#{@type}_#{@current_user.id}"
   end
 
-  def delete_comment_and_notifications(commentid, model_name)
-    Comment.find(commentid).destroy
-    public_uniqueid = "comment_on_#{model_name}_#{commentid}"
-    Notification.where(uniqueid: public_uniqueid).destroy_all
-    private_uniqueid = "comment_on_#{model_name}_private_#{commentid}"
-    Notification.where(uniqueid: private_uniqueid).destroy_all
+  def self.handle_members(args = {})
+    new(args).handle_members
   end
 
   # rubocop:disable MethodLength
-  def notifications_for_meeting_members(meeting, members, type)
-    uniqueid = "#{type}_#{current_user.id}"
+  def handle_members
     members.each do |member|
       next if member.user_id == current_user.id
```

Comment:
If the members are being passed by the caller to the service, you could _probably_ assume they won't contain the current user. In the event that they do, perhaps we should reject the current user in `initialize` instead of performing a `next` here. So something like:

```
def initialize(args)
  @members = args[:members].reject { |member| member.id == current_user.id }
  ...
end
```